src/libs/OptionsListUtils.js

```diff
@@ -96,6 +99,23 @@ function createOption(personalDetailList, report, draftComments, {showChatPrevie
         + _.unescape(report.lastMessageText)
         : '';
     const tooltipText = getReportParticipantsTitle(lodashGet(report, ['participants'], []));
+
+    // Using an object instead of an array since we can check if it has a key in constant time
```

Comment:
I'm not sure what "constant time" we're referring to here, maybe it would be better to plainly speak about the benefits are... e.g. is there some noticeable difference if we don't do this?

Reply:
`Set` allows us to check whether it has a value much faster than an array. An array requires iterating through each member, a Set can directly look up a value you provide it and tell you if it exists or not. If we had a `Set` vs. an `Array` with 10 values, the `Set` can be up to 10 times faster than the `Array`.

Reply:
There will be a noticeable difference if a user has a ton of reports to search through. Like I said, worst case it could be the difference between 0.2s and 1s.